src/TConsole.cpp

```diff
@@ -1719,10 +1719,13 @@ void TConsole::setHorizontalScrollBar(bool isEnabled)
     }
 }
 
-void TConsole::setScrolling(bool enabled)
+void TConsole::setScrolling(const bool state)
```

Comment:
I am sorry but "enabled" was not as clear as it might seem - especially when used below in `mScrollingEnabled = enabled;` - I felt that `state` worked better...!

Reply:
I agree with you.